packages/database/src/core/Repo.ts

```diff
@@ -653,7 +653,7 @@ export class Repo {
     forEach(stats, (stat: string, value: any) => {
       // pad stat names to be the same length (plus 2 extra spaces).
       for (let i = stat.length; i < longestName + 2; i++) stat += ' ';
-      console.log(stat + value);
+      log(stat + value);
```

Comment:
Again, this is actually meant to be logged to the console, without enabling any logging settings.

Reply:
SGTM